requirements/requirements.in

```diff
@@ -0,0 +1,3 @@
+flask
+flake8
```

Comment:
I recommend moving the `flake8` and `autopep8` deps into a separate `requirements-dev.txt` file. I also recommend you move the `requirements` files to the top-level. I have never seen a `requirements` directory in use before. It is a non-standard practice.